content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -610,17 +610,23 @@ plugins:
         values: ["cluster-services"]
 ```
 
-Now, "cluster-services" pods will be allowed in only those namespaces where a quota object with a matching `scopeSelector` is present.
-For example:
+Then, create a corresponding resource quota object in `kube-system` namespace:
+
+{{< codenew file="policy/priority-class-resourcequota.yaml" >}}
+
+```shell script
+$ kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system
```

Comment:
```suggestion
$ kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system
```

Add the code black only for `kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system`
you can refer to line 323-325 of this chapter

Reply:
Done.